src/lib/storage/chunk.hpp

```diff
@@ -78,6 +78,7 @@ class Chunk : private Noncopyable {
    *       call `get_segment again`, be aware that the returned object might have changed.
    */
   std::shared_ptr<AbstractSegment> get_segment(ColumnID column_id) const;
+  const Segments& get_segments() const;
```

Comment:
We're so inconsistent when naming getters with and without `get_` prefix. I would slightly prefer `segments()`, but not really a strong opinion.